basin3d/core/plugin.py

```diff
@@ -15,15 +15,15 @@
 
 
 """
-import logging
+from basin3d.core import monitor
 from types import MethodType
 from typing import Dict
 
 from basin3d.core.catalog import CatalogTinyDb
 from basin3d.core.models import DataSource
 from basin3d.core.schema.enum import FeatureTypeEnum
 
-logger = logging.getLogger(__name__)
+logger = monitor.get_logger(__name__)
```

Comment:
this module doesn't have any explicit logging calls. Do we still need to initialize the logger?

Reply:
Let's leave it since that is the way the code was already.  More than likey we will add logging in the module in the future or remove.